src/dropbox.ml

```diff
@@ -259,6 +290,10 @@ module type S = sig
 
   val media : t -> ?locale: string -> string -> link option Lwt.t
 
+  val shared_folders : ?shared_folder_id: string -> ?include_membership: bool
+                      -> t -> [ `Singleton of shared_folder
+                              | `List of shared_folders ] Lwt.t
```

Comment:
When shared_folder_id is set, it return a dictionary shared_folder. Otherwise, return a shared_folder list.
But we can indeed write `List of shared_folder list.